ql/src/java/org/apache/hadoop/hive/ql/parse/type/FunctionHelper.java

```diff
@@ -82,6 +82,20 @@ default RexNode foldExpression(RexNode expr) {
     return expr;
   }
 
+  boolean isAndFunction(FunctionInfo fi);
```

Comment:
Would you mind to add a short comment to all these methods, even if may seem obvious for some of them?